src/widgets/youtube/use-video-tracking.ts

```
@@ -0,0 +1,52 @@
+  video: VideoData;
+}
+
+export function useVideoTracking({ video, playerRef }: UseVideoTrackingProps) {
+  const trackingRef = useRef<NodeJS.Timeout | null>(null);
+  const isPlaying = useRef<boolean>(false);
+
+  function getPercentCompleted(node: YT.Player) {
+    const duration = node.getDuration() || 0;
+    const currentTime = node.getCurrentTime() || 0;
+    const percentCompleted = duration
+      ? parseInt(((currentTime / duration) * 100).toFixed(2), 10)
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)